notebooks/openvino/vision_language_quantization.ipynb

```diff
@@ -397,6 +481,44 @@
     "q_model.save_pretrained(int8_model_path)"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c9cf1523",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Total size: 248.86 MB\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(f\"Total size: {get_model_size(base=\"smolvlm_int8\") / (1024**2):.2f} MB\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "id": "26883f9c",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Average Inference latency: 2.1692 seconds\n"
+     ]
+    }
+   ],
+   "source": [
+    "# 2 rounds of warm-up and 5 rounds of inference (to measure the average)\n",
+    "avg_latency = elapsed_time(q_model, inputs, nb_pass=5, warmup=2)\n",
+    "print(f\"Average Inference latency: {avg_latency:.4f} seconds\")"
+   ]
+  },
```

Comment:
failing tests coming from `get_model_size` and `elapsed_time`  being defined later, I think we can remove @ezelanza 

```suggestion
```